ui/js/signup.js

```diff
@@ -299,7 +301,13 @@ require(['jquery', 'underscore', 'oae.core', 'iso3166'], function($, _, oae, iso
                 },
                 'emaildomain': {
                     'method': oae.api.util.validation().isValidEmailDomainForTenant,
-                    'text': oae.api.i18n.translate('__MSG__YOU_CAN_ONLY_REGISTER_WITH_A_TENANT_EMAIL_ADDRESS__', null, {'emailDomain': tenantEmailDomain})
+                    'text': function() {
+                        if (tenantEmailDomains.length === 1) {
+                            return oae.api.i18n.translate('__MSG__YOU_CAN_ONLY_REGISTER_WITH_AN_EMAIL_ADDRESS_ENDING_IN__', null, {'emailDomain': tenantEmailDomains[0]});
+                        } else {
+                            return oae.api.i18n.translate('__MSG__YOU_MUST_HAVE_AN_EMAIL_ADDRESS_THAT_ENDS_IN_ONE_OF__', null, {'emailDomains': tenantEmailDomains.join(', ')});
```

Comment:
Do these messages have to have such different wording? I feel like they can be consistent:

`YOU_CAN_ONLY_REGISTER_WITH_AN_EMAIL_ADDRESS_ENDING_IN_` and 
`YOU_CAN_ONLY_REGISTER_WITH_AN_EMAIL_ADDRESS_ENDING_IN_ONE_OF_`

or

`YOU_MUST_HAVE_AN_EMAIL_ADDRESS_THAT_ENDS_IN_` and 
`YOU_MUST_HAVE_AN_EMAIL_ADDRESS_THAT_ENDS_IN_ONE_OF`